cmd/dump/dump.go

```
@@ -40,17 +43,15 @@ func main() {
 	for {
 		record := iterator.Record()
 
-		keyReader, keySize := record.Key()
-		key := make([]byte, keySize)
-		if _, err = keyReader.Read(key); err != nil {
-			log.Fatal(err)
-		}
+		keyReader, _ := record.Key()
+		keyBuf := new(bytes.Buffer)
+		io.Copy(keyBuf, keyReader)
+		key := keyBuf.Bytes()
 
-		valReader, valSize := record.Value()
-		value := make([]byte, valSize)
-		if _, err = valReader.Read(value); err != nil {
-			log.Fatal(err)
-		}
+		valReader, _ := record.Value()
+		valBuf := new(bytes.Buffer)
+		io.Copy(valBuf, valReader)
```

It's a bad practice to ignore errors